artifact-reftype-spec.md

```diff
@@ -0,0 +1,78 @@
+# OCI Artifact Manifest Spec (Phase-1 Reference Types)
+
+The OCI artifact manifest generalizes the use of [OCI image manifest][oci-image-manifest-spec]. It provides a means to define a wide range of artifacts, including a chain of related artifacts enabling SBoMs, on-demand loading, Signatures and metadata. 
+
+To enable a fall 2021 focus on supply chain security,  **Phase 1** will narrowly focus on Reference Type support, giving time for further generalization with less time constraints.
+
+For usage and scenarios, see [artifact-manifest.md](./artifact-manifest.md)
+
+## Example OCI Artifact Manifests
+
+The following are Phase 1 examples:
+
+- [`net-monitor:v1` oci container image](./artifact-manifest/net-monitor-oci-image.json)
+- [`net-monitor:v1` notary v2 signature](./artifact-manifest/net-monitor-image-signature.json)
+- [`net-monitor:v1` sample sbom](./artifact-manifest/net-monitor-image-sbom.json)
+- [`net-monitor:v1` nydus image with on-demand loading](./artifact-manifest/net-monitor-image-nydus-ondemand-loading.json)
+
+## OCI Artifact Manifest Properties
+
+For **Phase 1**, an artifact manifest provides a collection of blobs and a reference to a manifest of another artifact.
+
+- **`schemaVersion`** *int*
+
+  This REQUIRED property specifies the artifact manifest schema version.
+  For this version of the specification, this MUST be `3`. The value of this field WILL change as the manifest schema evolves. Minor version changes to the `oci.artifact.manifest` spec MUST be additive, while major version changes MAY be breaking. Artifact clients MUST implement version checking to allow for future, yet unknown changes. Artifact clients MUST ignore additive properties to minor versions. Artifact clients MAY support major changes, with no guarantee major changes MAY impose breaking changing behaviors. Artifact authors MAY support new and older schemaVersions to provide the best user experience.
+
+- **`mediaType`** *string*
+
+  This field contains the `mediaType` of this document, differentiating from [image-manifest][oci-image-manifest-spec] and [oci-image-index]. The mediaType for this manifest type MUST be `application/vnd.oci.artifact.manifest.v1-rc1+json`, where the version WILL change to reflect newer versions. `v1-rc1` indicates a stable release, and a step towards a completed `v1` spec. Artifact authors SHOULD support multiple `mediaType` versions to provide the best user experience for their artifact type.
```

Comment:
The mediaType is declared as `application/vnd.oci.artifact.manifest.v1-rc1+json` whereas the examples use `application/vnd.oci.artifact.reftype.manifest.v1+json`

Reply:
Thanks, @corburn, I've got a few updates queued up that will cleanup the inconsistencies. While we may still change the `mediaType` string, it should be consistent across the references for the PR.

Reply:
@coburn, please see latest updates.

Reply:
@SteveLasker looks great!